I still get one test failure from running the tests twice

```
Traceback (most recent call last):
  File "/Users/aaronmeurer/Dropbox/Documents/Python/sympy/sympy/sympy/polys/tests/test_rootoftools.py", line 525, in test_eval_approx_relative
    -21/220, 15/256 - 805*I/256, 15/256 + 805*I/256]
AssertionError
```

--- a/sympy/polys/rootisolation.py
+++ b/sympy/polys/rootisolation.py
@@ -1794,7 +1794,7 @@ class ComplexInterval(object):
 
     >>> from sympy import CRootOf, Rational, S
     >>> from sympy.abc import x
-    >>> CRootOf.clear_cache()
+    >>> CRootOf.clear_cache()  # for doctest reproducibility
     >>> root = CRootOf(x**10 - 2*x + 3, 9)
     >>> i = root._get_interval(); i
     (3/64, 3/32) x (9/8, 75/64)

--- a/sympy/polys/rootoftools.py
+++ b/sympy/polys/rootoftools.py
@@ -188,7 +188,7 @@ class ComplexRootOf(RootOf):
     CRootOf is a way to reference a particular root of a
     polynomial. If there is a rational root, it will be returned:
 
-    >>> CRootOf.clear_cache()
+    >>> CRootOf.clear_cache()  # for doctest reproducibility
     >>> CRootOf(x**2 - 4, 0)
     -2
 
@@ -726,6 +726,17 @@ class ComplexRootOf(RootOf):
 
     @classmethod
     def clear_cache(cls):
+        """Reset cache for reals and complexes.
+
+        The intervals used to approximate a root instance are updated
+        as needed. When a request is made to see the intervals, the
+        most current values are shown. `clear_cache` will reset all
+        CRootOf instances back to their original state.
+
+        See Also
+        ========
+        _reset
+        """
         global _reals_cache, _complexes_cache
         _reals_cache = _pure_key_dict()
         _complexes_cache = _pure_key_dict()

--- a/sympy/polys/tests/test_rootoftools.py
+++ b/sympy/polys/tests/test_rootoftools.py
@@ -520,6 +520,7 @@ def test_pure_key_dict():
 
 
 def test_eval_approx_relative():
+    CRootOf.clear_cache()
     t = [CRootOf(x**3 + 10*x + 1, i) for i in range(3)]
     assert [i.eval_rational(1e-1) for i in t] == [
         -21/220, 15/256 - 805*I/256, 15/256 + 805*I/256]

